components/gitpod-db/src/typeorm/migration/1662040283793-AddUsageTable.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddUsageTable1662040283793 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `CREATE TABLE \`d_b_usage\` (
+                \`id\` char(36) NOT NULL,
+                \`attributionId\` varchar(255) NOT NULL,
+                \`description\` varchar(255) NOT NULL,
+                \`creditCents\` bigint NOT NULL,
+                \`effectiveTime\` varchar(255) NOT NULL,
+                \`creationTime\` varchar(255) NOT NULL,
+                \`kind\` varchar(255) NOT NULL,
+                \`workspaceInstanceId\` char(36) NULL,
+                \`draft\` BOOLEAN NOT NULL,
+                \`metadata\` text NULL,
+                \`_lastModified\` timestamp(6) NOT NULL,
+
+                INDEX \`IDX_usage__attribution_id\` (\`attributionId\`),
```

Comment:
Judgind from [here](https://github.com/gitpod-io/gitpod/pull/12584/files#diff-875a57097d6d33f86f7738c41354760bcec938dd43ac5d3d3c96eba044d0b37dR44), we should also have an index:

```suggestion
                INDEX \`IDX_usage__attribution_id\` (\`attributionId\`),
                INDEX \`IDX_usage_attribution_id_effectiveTime\` (\`attributionId\`, \`effectiveTime\`),
```

Reply:
I have added one on `effectiveTime` only. I'm not super sure about our use cases yet, though.